### ISSUE TITLE
documentPictureInPicture bypasses snow

### ISSUE THREAD

**Issue by NDevTK** [LavaMoat/snow#77]
A new feature exists in chromium under an origin trial https://chromestatus.com/feature/5755179560337408
Probably should be treated like a popup.
```js
onclick = async () => {
    w = await documentPictureInPicture.requestWindow();
    w.alert.call(window, 'hello world');
    w.close();
}
```

**Comment by weizman** [comment]
Thanks for the heads up! 
Will look in to that soon 🙏

**Comment by NDevTK** [comment]
At the moment easiest way to test it is with chrome://flags/#document-picture-in-picture-api

**Comment by weizman** [comment]
Here's a draft. harder than i thought. still need to write the tests. #79

**Comment by weizman** [comment]
merged fix #79

**Comment by weizman** [comment]
Lol now i see the connection. Awesome job man!! https://twitter.com/bugschromium/status/1681366231293603840?s=46&t=3av-wxCFEbML6grYvIm18A @NDevTK
